lib/api/build.rs

```diff
@@ -135,6 +135,7 @@ fn configure_validation(builder: Builder) -> Builder {
             ("SetPayloadPoints.collection_name", "length(min = 1, max = 255)"),
             ("DeletePayloadPoints.collection_name", "length(min = 1, max = 255)"),
             ("ClearPayloadPoints.collection_name", "length(min = 1, max = 255)"),
+            ("UpdateBatchPoints.collection_name", "length(min = 1, max = 255)"),
```

Comment:
I'll look into it. Other batch endpoints have a `("<name>", "")` validation annotation, so that might be a trick that does it, but it's easy to test anyway, so easy to try different things.

Reply:
The second `""` is transformed into `#[validate]` while `"something"` would be translated into `#[validate(something)]`.

This is [validator](https://docs.rs/validator/latest/validator/) syntax, which we use throughout the crate.

Something like `"length(min = 1)"` would normally do the trick, but there are some `Validate`/`Serialize` trait impl errors.

Reply:
@timvisee I implemented the "nuclear option" of deriving `Serialize` on all message types, which allows us to just derive `Validate` for the operations. On the one hand this is nice, because it allows us to easily derive `Validate` for any other message type later, on the other hand, we lose some conveniences on `prost-types` types that are currently not used. I don't have a particular preference for doing it one way or the other. What do you think?